Kernel/Devices/HID/USBMouseDevice.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Try.h>
+#include <Kernel/Bus/USB/USBDescriptors.h>
+#include <Kernel/Bus/USB/USBRequest.h>
+#include <Kernel/Devices/DeviceManagement.h>
+#include <Kernel/Devices/HID/USBMouseDevice.h>
+#include <Kernel/KString.h>
+
+namespace Kernel {
+
+ErrorOr<NonnullRefPtr<USBMouseDevice>> USBMouseDevice::try_to_initialize(USB::Device const& usb_device)
```

Comment:
This should probably be called `try_create`, as it's _not_ initialising the `USB::Device` passed in, but rather creating a `USBMouseDevice` _with_ it.